src/app/loaders/express/__tests__/helmet.spec.ts

```diff
@@ -31,17 +31,17 @@ describe('helmetMiddlewares', () => {
     fontSrc: ["'self'", 'data:', 'https://fonts.gstatic.com/'],
     scriptSrc: [
       "'self'",
-      'https://www.googletagmanager.com/',
       'https://ssl.google-analytics.com/',
       'https://www.google-analytics.com/',
       'https://www.tagmanager.google.com/',
       'https://www.google.com/recaptcha/',
       'https://www.recaptcha.net/recaptcha/',
-      'https://www.gstatic.com/recaptcha/',
+      'https://www.gstatic.com/recaptcha/releases/',
```

Comment:
Not sure if we can be even more granular and do something like this: 

```suggestion
      'https://www.gstatic.com/recaptcha/releases/*/recaptcha__en.js',
```

Reply:
The part after `releases/` seems to be some kinda ID identifying the release, and it's the same between prod and staging, so maybe we could hard-code that? Might want to see where and when it is generated before doing so though.

Reply:
I actually considered hardcoding that but was worried about third party changing the specific file names.

Reply:
Okay! Let's keep to this then, just in case.